src/components/Global.astro

```diff
@@ -1,51 +1,52 @@
 <style global>
 :root {
-    --main: #262260;
-    --secondary: #f55a8a;
-    --tertiary: #7de5e5;
-    --dark: #1a1229;
-    --white: #ffffff;
-    --mainfont: 'main', sans-serif;
+	--main: #262260;
+	--secondary: #f55a8a;
+	--tertiary: #7de5e5;
+	--dark: #1a1229;
+	--white: #ffffff;
+	--black: #000000;
```

Comment:
Other than the indentation change, I've added this `--black` value. I've only come across one use of it so far, but best to consolidate all our colors together.